Azure/sonic-utilities
# [counters] Keep counters cache in a single directory

#### What I did

To fix https://github.com/Azure/sonic-buildimage/issues/9817. Cache all counters in the same place. Created a UserCache helper class to access the cache directory.

#### How I did it

Implemented UserCache class. Changed all stats commands to use new class. Adopted fast-reboot script.

#### How to verify it

Run on the switch and verify counters stats command and clear commands work correctly. After config reload or cold/fast reboot counters cache is removed.

#### Previous command output (if the output of a command-line utility has changed)

#### New command output (if the output of a command-line utility has changed)
